### PR TITLE
Allow setting of dynamodb endpoint url

### PR DESCRIPTION
**Pull request by jaustinpage**
Amazon provides [DynamoDB for local dev](https://aws.amazon.com/blogs/aws/dynamodb-local-for-desktop-development/). In order to use this with a local flask app, you would need to set the [boto3.session.Session.client endpoint_url](http://boto3.readthedocs.io/en/latest/reference/core/session.html#boto3.session.Session.client), but presently an option to do this with flask-sessionstore is not provided. This adds such an option.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -110,59 +110,62 @@ to control your session lifetime.**
 A list of configuration keys also understood by the extension:
 
 ============================= ==============================================
-``SESSION_TYPE``              Specifies which type of session interface to
```
Comment (jaustinpage) [inline docs/index.rst]: Sorry for the huge line of changes, this is just to set the whitespace consistently...

File: docs/index.rst [hunk 2]
```diff
@@ -110,59 +110,62 @@ to control your session lifetime.**
...
+                                     or in the local aws config.
+``SESSION_DYNAMODB_KEY_ID``          The AWS key id for connecting to Dynamo. Uses environment
+                                     variable or local config if not set.
+``SESSION_DYNAMODB_SECRET``          The AWS secret access key for connecting to Dynamo. Uses
+                                     environment variable or local config if not set.
+``SESSION_DYNAMODB_REGION``          The region where the dynamodb table is located. Uses
+                                     environment variable or the local config if not set.
+``SESSION_DYNAMODB_TABLE``           The name of the table in DyanmoDB to store session
+                                     data. Default is "sessions".
+``SESSION_DYNAMODB_ENDPOINT_URL``    The endpoint url for connecting to Dynamo. Uses
```
Comment (jaustinpage) [inline docs/index.rst]: These 3 lines, [R166](https://github.com/mcrowson/flask-sessionstore/pull/20/files#diff-caf2a6b8f4947d018f68893c695b5202R166)-[R168](https://github.com/mcrowson/flask-sessionstore/pull/20/files#diff-caf2a6b8f4947d018f68893c695b5202R168) are the only "real" changes here.

File: docs/index.rst (hunk 1 above)
Comment (jaustinpage) [inline docs/index.rst]: I put the whitespace change in a separate commit for clarity.

**Comment by mcrowson** [comment]
Looks great!
